### DISCUSSION TITLE
Is conditional load possible with Pydantic V2 model_validate() ?

### DISCUSSION THREAD

**Discussion by jordantshaw** [pydantic/pydantic#6861] [Question]
With the release of pydantic v2, is it possible to load a model and exclude certain fields when loading using the new `model_validate` method? I was hoping the new `context` parameter was able to achieve this, but it looks this does not quite do what I was thinking.

My use case is very similar to this issue [#1806]( https://github.com/pydantic/pydantic/issues/1806), when loading models from an ORM object, if there are related objects within the main ORM object, then they would be lazily loaded.  I want to be be able to exclude certain fields from being lazily loaded.  

Is there currently any possible way to do this within pydantic either through `model_validate` or any other method?

**Comment by braindevices** [comment]
please provide a full working code and describe the difference of the actually result and your expected result.

It is not clear what you want. exclude fields during validate does not make much sense at all.
The model by default ignore undefined fields anyway.
If you do have the fields defined, then there is no point to not valid those fields.

the include/exclude only a concept during export/dump/ser but not during validation.

**Reply by jordantshaw** [reply]
> It is not clear what you want. exclude fields during validate does not make much sense at all.

There is actually a pretty big use case for this by anyone who is creating pydantic models from ORM objects.  If you have related ORM objects that are lazily loaded, the pydnatic model will cause these relationships to load, thus triggering another SQL query to the database.  

It would be very convenient to have an option in pydantic to prevent this, like perhaps a maximum depth to validate, or an option to exclude certain fields when loading the pydantic model. 

> 
> The model by default ignore undefined fields anyway.
> If you do have the fields defined, then there is no point to not valid those fields.

If the field is not defined, then you would want the the default value of that field to be used.  However, this currently does not work when loading pydantic models from ORM objects.  Because instead of leaving the field unset  it triggers a lazy load from the ORM object, and causes that field to become set.
